modules/effects/arrow-effects-rx2-extensions/src/main/kotlin/arrow/effects/rx2/extensions/observablek.kt

```diff
@@ -10,6 +10,14 @@ import arrow.effects.rx2.extensions.observablek.monad.monad
 import arrow.effects.rx2.extensions.observablek.monadError.monadError
 import arrow.effects.rx2.fix
 import arrow.effects.typeclasses.*
+import arrow.effects.typeclasses.Async
```

Comment:
Are these necessary when we already have `import arrow.effects.typeclasses.*` in line 12?